samples/java/appengine-java8/README.md

```diff
@@ -0,0 +1,89 @@
+# Snapshot Debugger Examples for Google App Engine Standard Environment for Java 8
```

Comment:
It's a bit strange to have this README and the helloworld/README given the amount of overlap.  While the helloworld/README was direct and helpful for getting the sample going, this one *almost* was useful on its own.  

It's possible to resolve this weirdness with a simple foreword indicating that this README provides general information about Snapshot Debugger & GAE Java8 and then direct linking the instructions for the sample itself.  Otherwise users are going to run the risk of landing on this README and being unable to proceed.

Reply:
Done.